### DISCUSSION TITLE
I had to change some strings

### DISCUSSION THREAD

**Discussion by kebekus** [Akaflieg-Freiburg/enrouteText#44] [Team Posts]
Dear Translators,

Regretfully, I had to change some of the new strings you had already translated. I realized too late that the translations (which tend to be longer than the original English strings) do not fit the display on small devices. Sorry for that. I hope you are not too upset with me.

Best,

Stefan.

**Comment by TTC70** [comment]
Hi Stefan,
no problem at all.
Please let me know, now or in the future, which are the strings which need to be reduced. I will try to find a shorter different way to translate the same concept.
enjoy.
